components/clientComponents/globals/Header/Header.tsx

```diff
@@ -26,7 +28,27 @@ export const Header = ({ context = "default", className }: HeaderParams) => {
     i18n: { language },
   } = useTranslation(["common", "form-builder", "admin-login"]);
 
-  const isBannerEnabled = t("campaignBanner.enabled");
+  const [isBannerEnabled, setBannerData] = useState(false);
+  const [bannerType, setBannerType] = useState("");
+  const [bannerMessage, setBannerMessage] = useState("");
+
+  useEffect(() => {
+    async function fetchBannerData() {
+      const isEnabled = t("campaignBanner.enabled") === "true";
```

Comment:
Maybe update this to isBannerEnabled --- little easier to figure out what's enabled here